docs/development/migration.rst

```diff
@@ -0,0 +1,49 @@
+.. _migration_index:
+
+Migration
```

Comment:
I initially read this as documentation for devops, but looking at the lack of docs for Open Klant, I don't think any client/devops will find these docs.

Do we need to document/mention this command in the changelog to make sure at least Sjoerd knows about it?

Reply:
I moved the generic part of this file to the user targeted documentation (under `installation`) and also updated the changelog.